api_spec.yaml

```diff
@@ -1752,6 +1760,81 @@ paths:
                                 type: string
                                 enum:
                                     - Unauthorised access
+    /dashboard/api/tenants/third-party/boxy-saml:
+        put:
+            tags:
+                - Multitenancy
+            summary: Create or update third party config for a tenant with BoxyHQ
```

Comment:
Create or Update client of Boxy SAML provider using SAML xml ? - rephrase it so that it makes sense

Reply:
Updated